Jellyfin.Server.Implementations/Activity/ActivityManager.cs

```diff
@@ -40,8 +40,9 @@ public void Create(ActivityLog entry)
         /// <inheritdoc/>
         public async Task CreateAsync(ActivityLog entry)
         {
-            using var dbContext = _provider.CreateContext();
-            await dbContext.ActivityLogs.AddAsync(entry);
+            await using var dbContext = _provider.CreateContext();
```

Comment:
@Bond-009 What about `ConfigureAwait` here?

Reply:
Good point.
Should be done like this:
```cs
var dbContext = _provider.CreateContext();
await using dbContext.ConfigureAwait(false);
```

Reply:
That doesn't compile

Reply:
IK, blame microsoft
https://docs.microsoft.com/en-us/dotnet/standard/garbage-collection/implementing-disposeasync#stacked-usings